.github/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+<!-- 
+    Thank you for your contribution to the polyPod repo. ♡
+
+    Please, make sure that:
+    - Your code builds clean without any errors or warnings (You've run `./build.js` locally)
+    - Tests for the changes have been added, if possible (for bug fixes / new features)
+    - Docs have been reviewed and added / updated, if needed (for bug fixes / new features)
+    - Commits in this PR are minimal and [have descriptive commit messages.](<https://cbea.ms/git-commit/>) <https://cbea.ms/git-commit/>
+
+    Before submitting this PR, please fill the following information on the following sections.
+-->
+
+
+# Description
+<!-- 
+    Please, include a summary of the changes you made here and the related issue.
+    Some questions to help you give more context:
+     - What is the current behavior?
+     - Why we need this change/addition?
+     - What is the new behavior (if this a feature change)?
+     - What should we expect after this change/addition? 
+     - How did you test it?
+     - Does this introduce a breaking change?
+     - Are there any dependencies that are required for this change?
+ -->
+
+<!-- Link here the JIRA issue in case of a bug / feature -->
+### Fixes #(jira-issue-number)
+
+
+## Other information
+<!-- 
+    Any other information that is important to this PR. 
+    e.g. if regards a frontend UI change, add some screenshots 
+    of how the app looked before and how it looks after the change.
+-->
+
+
+♡ Thank you!
```

Comment:
I love the emoji!
(more should be mandatory)

Reply:
...you've 5 more! 
(merge before Felix comes back 😆 )